src/server/pages/home-page.jsx

```diff
@@ -24,15 +29,67 @@ const LetsGetStarted = () => (
 
 export default class HomePage extends React.Component {
   render() {
+    const isLoggedIn = 'name' in this.props.userInfo
     const { notebookList } = this.props
     return (
       <div>
         <Header userInfo={this.props.userInfo} />
         <PageBody>
-          {notebookList.length ?
-            <TrendingNotebooksPage notebookList={notebookList} /> :
-            <LetsGetStarted />
+          <TopContainer>
+            {!isLoggedIn && <MarketingCopySplash /> }
+            {isLoggedIn && <LoggedInSplash userInfo={this.props.userInfo} />}
+            <PageHeader>Try These Examples</PageHeader>
```

Comment:
I am not sure, but I suspect having the user notebooks + public examples + firehose seems like a lot of data when logged in. Probably not the end of the world though.